lib/sprockets/server.rb

```diff
@@ -23,7 +23,7 @@ def call(env)
       start_time = Time.now.to_f
       time_elapsed = lambda { ((Time.now.to_f - start_time) * 1000).to_i }
 
-      if env['REQUEST_METHOD'] != 'GET'
+      if !['GET', 'HEAD'].include?(env['REQUEST_METHOD'])
```

Comment:
That may well be the case (don't know where you got the 60% from, however) – the Sprockets server is used only in development on your own machine and I seriously doubt that it is, in fact, not a micro optimization.